### PR TITLE
Url camelcase fix

### LINKED ISSUE #254: [Bug] Use camelcase in prop names
## Description

These components use the string 'URL' in the prop name.

https://github.com/WordPress/openverse-frontend/blob/428be354eada6b29dc3d63f21c7960e42386b423/src/components/ImageDetails/ImageInfo.vue#L65
https://github.com/WordPress/openverse-frontend/blob/428be354eada6b29dc3d63f21c7960e42386b423/src/components/ImageDetails/ImageAttribution.vue#L24
https://github.com/WordPress/openverse-frontend/blob/428be354eada6b29dc3d63f21c7960e42386b423/src/components/MediaInfo/MediaLicense.vue#L72
https://github.com/WordPress/openverse-frontend/blob/428be354eada6b29dc3d63f21c7960e42386b423/src/components/MediaInfo/CopyLicense.vue#L188

This leads to the following syntax where the component is used with props:

```html
<Component :something-something-u-r-l="value"/>
```

This is not looking good. Changing the props from 'URL' to 'Url' would fix this.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by theshakeabhi**
## Fixes
Fixes #254   by @dhruvkb 

## Description
This PR would make the prop name uniform across the code base by changing the non-camelcase to camel case.
For now these changes has been made across the code base:
`ccLicenseURL to ccLicenseUrl`
`licenseURL to licenseUrl`
`openverseLicenseURL to openverseLicenseUrl`

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/ImageDetails/ImageAttribution.vue [hunk 1]
```diff
@@ -21,13 +21,13 @@ export default {
   props: {
     id: {},
     image: {},
-    ccLicenseURL: String,
+    ccLicenseUrl: String,
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:24]:
```suggestion
    openverseLicenseUrl: String,
```

File: src/components/ImageDetails/ImageAttribution.vue [hunk 2]
```diff
@@ -21,13 +21,13 @@ export default {
...
     fullLicenseName: String,
     attributionHtml: String,
   },
   computed: {
-    licenseURL() {
-      return `${this.ccLicenseURL}&atype=rich`
+    licenseUrl() {
+      return `${this.ccLicenseUrl}&atype=rich`
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:30]:
```suggestion
      return `${this.openverseLicenseUrl}&atype=rich`
```

File: src/components/ImageDetails/ImageInfo.vue [hunk 3]
```diff
@@ -63,7 +63,7 @@ export default {
   },
   props: [
     'image',
-    'ccLicenseURL',
+    'ccLicenseUrl',
```
Comment (obulat) [inline src/components/ImageDetails/ImageInfo.vue:66]:
```suggestion
    'openverseLicenseUrl',
```

File: src/components/ImageDetails/PhotoDetails.vue [hunk 4]
```diff
@@ -126,7 +126,7 @@
           <ImageAttribution
             data-testid="image-attribution"
             :image="image"
-            :cc-license-u-r-l="openverseLicenseURL"
+            :cc-license-u-r-l="openverseLicenseUrl"
```
Comment (obulat) [inline src/components/ImageDetails/PhotoDetails.vue:129]:
```suggestion
            :openverse-license-url="openverseLicenseUrl"
```

File: src/components/ImageDetails/PhotoDetails.vue [hunk 5]
```diff
@@ -141,7 +141,7 @@
           <ImageInfo
             data-testid="image-info"
             :image="image"
-            :cc-license-u-r-l="openverseLicenseURL"
+            :cc-license-u-r-l="openverseLicenseUrl"
```
Comment (obulat) [inline src/components/ImageDetails/PhotoDetails.vue:144]:
```suggestion
            :openverse-license-url="openverseLicenseUrl"
```

File: src/components/MediaInfo/MediaReuse.vue [hunk 6]
```diff
@@ -5,13 +5,13 @@
       <MediaLicense
         :license="media.license"
         :full-license-name="fullLicenseName"
-        :license-u-r-l="licenseURL"
+        :license-u-r-l="licenseUrl"
```
Comment (obulat) [inline src/components/MediaInfo/MediaReuse.vue:8]:
```suggestion
        :license-url="licenseUrl"
```

File: src/components/MediaInfo/MediaReuse.vue [hunk 7]
```diff
@@ -5,13 +5,13 @@
...
         class="column"
       />
       <CopyLicense
         :media="media"
         :full-license-name="fullLicenseName"
-        :license-u-r-l="licenseURL"
+        :license-u-r-l="licenseUrl"
```
Comment (obulat) [inline src/components/MediaInfo/MediaReuse.vue:14]:
```suggestion
        :license-url="licenseUrl"
```

File: src/pages/audio/_id.vue [hunk 8]
```diff
@@ -4,7 +4,7 @@
     <MediaReuse
       data-testid="audio-attribution"
       :media="audio"
-      :license-u-r-l="openverseLicenseURL"
+      :license-u-r-l="openverseLicenseUrl"
```
Comment (obulat) [inline src/pages/audio/_id.vue:7]:
```suggestion
      :license-url="openverseLicenseUrl"
```

File: src/utils/attribution-html.js [hunk 9]
```diff
@@ -1,4 +1,4 @@
-function attributionHtml(image, ccLicenseURL, fullLicenseName) {
+function attributionHtml(image, ccLicenseUrl, fullLicenseName) {
```
Comment (obulat) [inline src/utils/attribution-html.js:1]:
```suggestion
function attributionHtml(image, openverseLicenseUrl, fullLicenseName) {
```

File: src/utils/attribution-html.js [hunk 10]
```diff
@@ -14,7 +14,7 @@ function attributionHtml(image, ccLicenseURL, fullLicenseName) {
   } else if (image.creator && !image.creator_url) {
     creator = `<span> by <span>${image.creator}</span></span>`
   }
-  const licenseLink = ` is licensed under <a href="${ccLicenseURL}" style="margin-right: 5px;">${fullLicenseName.toUpperCase()}</a>`
+  const licenseLink = ` is licensed under <a href="${ccLicenseUrl}" style="margin-right: 5px;">${fullLicenseName.toUpperCase()}</a>`
```
Comment (obulat) [inline src/utils/attribution-html.js:17]:
```suggestion
  const licenseLink = ` is licensed under <a href="${openverseLicenseUrl}" style="margin-right: 5px;">${fullLicenseName.toUpperCase()}</a>`
```

File: src/utils/attribution-html.js [hunk 11]
```diff
@@ -27,7 +27,7 @@ function attributionHtml(image, ccLicenseURL, fullLicenseName) {
       .join('')
   }
 
-  const licenseImgLink = `<a href="${ccLicenseURL}" target="_blank" rel="noopener noreferrer" style="display: inline-block;white-space: none;margin-top: 2px;margin-left: 3px;height: 22px !important;">${licenseIcons}</a>`
+  const licenseImgLink = `<a href="${ccLicenseUrl}" target="_blank" rel="noopener noreferrer" style="display: inline-block;white-space: none;margin-top: 2px;margin-left: 3px;height: 22px !important;">${licenseIcons}</a>`
```
Comment (obulat) [inline src/utils/attribution-html.js:30]:
```suggestion
  const licenseImgLink = `<a href="${openverseLicenseUrl}" target="_blank" rel="noopener noreferrer" style="display: inline-block;white-space: none;margin-top: 2px;margin-left: 3px;height: 22px !important;">${licenseIcons}</a>`
```

File: test/unit/specs/components/ImageDetails/copy-license.spec.js [hunk 12]
```diff
@@ -37,7 +37,7 @@ describe('CopyLicense', () => {
         creator: 'John',
         creator_url: 'http://creator.com',
       },
-      ccLicenseURL: 'http://license.com',
+      ccLicenseUrl: 'http://license.com',
```
Comment (obulat) [inline test/unit/specs/components/ImageDetails/copy-license.spec.js:40]:
```suggestion
      openverseLicenseUrl: 'http://license.com',
```

File: test/unit/specs/components/ImageDetails/image-attribution.spec.js [hunk 13]
```diff
@@ -21,7 +21,7 @@ describe('ImageAttribution', () => {
         creator: 'John',
         creator_url: 'http://creator.com',
       },
-      ccLicenseURL: 'http://license.com',
+      ccLicenseUrl: 'http://license.com',
```
Comment (obulat) [inline test/unit/specs/components/ImageDetails/image-attribution.spec.js:24]:
```suggestion
      openverseLicenseUrl: 'http://license.com',
```

File: test/unit/specs/components/ImageDetails/image-info.spec.js [hunk 14]
```diff
@@ -21,7 +21,7 @@ describe('Image Info', () => {
         creator: 'John',
         creator_url: 'http://creator.com',
       },
-      ccLicenseURL: 'http://license.com',
+      ccLicenseUrl: 'http://license.com',
```
Comment (obulat) [inline test/unit/specs/components/ImageDetails/image-info.spec.js:24]:
```suggestion
      openverseLicenseUrl: 'http://license.com',
```

File: src/assets/fonts/VocabularyIcons.json [hunk 15]
```diff
@@ -799,7 +799,7 @@
         "designer": "Creative Commons",
         "designerURL": "https://opensource.creativecommons.org",
         "license": "MIT",
-        "licenseURL": "https://opensource.org/licenses/MIT"
+        "licenseUrl": "https://opensource.org/licenses/MIT"
```
Comment (obulat) [inline src/assets/fonts/VocabularyIcons.json:802]: We don't need to change anything in this file.

**Review by obulat — CHANGES_REQUESTED** [review]
Thank you for your PR!
There are several changes I've added inline: it's better to finish renaming `ccLicenseUrl` to `openverseLicenseUrl`, and rename prop names, as well as values.

File: src/components/ImageDetails/ImageAttribution.vue [hunk 16]
```diff
@@ -3,12 +3,12 @@
...
       :full-license-name="fullLicenseName"
       :license="image.license"
-      :license-u-r-l="licenseURL"
+      :license-u-r-l="licenseUrl"
     />
     <CopyLicense
       :media="image"
       :full-license-name="fullLicenseName"
-      :license-u-r-l="licenseURL"
+      :license-u-r-l="licenseUrl"
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:11]:
```suggestion
      :license-url="licenseUrl"
```

File: src/components/ImageDetails/ImageAttribution.vue [hunk 17]
```diff
@@ -3,12 +3,12 @@
     <MediaLicense
       :full-license-name="fullLicenseName"
       :license="image.license"
-      :license-u-r-l="licenseURL"
+      :license-u-r-l="licenseUrl"
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:6]:
```suggestion
      :license-url="licenseUrl"
```

File: src/assets/fonts/VocabularyIcons.json (hunk 15 above)
Comment (obulat) [reply src/assets/fonts/VocabularyIcons.json:802]:
```suggestion
        "licenseURL": "https://opensource.org/licenses/MIT"
```

File: src/assets/fonts/VocabularyIcons.json [hunk 18]
```diff
@@ -818,7 +818,7 @@
             "designer": "Creative Commons",
             "designerURL": "https://opensource.creativecommons.org",
             "license": "MIT",
-            "licenseURL": "https://opensource.org/licenses/MIT"
+            "licenseUrl": "https://opensource.org/licenses/MIT"
```
Comment (obulat) [inline src/assets/fonts/VocabularyIcons.json:821]:
```suggestion
            "licenseURL": "https://opensource.org/licenses/MIT"
```

File: src/assets/fonts/VocabularyIcons.json [hunk 19]
```diff
@@ -1447,7 +1447,7 @@
         "designer": "Creative Commons",
         "designerURL": "https://opensource.creativecommons.org",
         "license": "MIT",
-        "licenseURL": "https://opensource.org/licenses/MIT"
+        "licenseUrl": "https://opensource.org/licenses/MIT"
```
Comment (obulat) [inline src/assets/fonts/VocabularyIcons.json:1450]:
```suggestion
        "licenseURL": "https://opensource.org/licenses/MIT"
```

File: src/components/ImageDetails/ImageAttribution.vue [hunk 20]
```diff
@@ -21,13 +21,13 @@ export default {
   props: {
     id: {},
     image: {},
-    ccLicenseURL: String,
+    openverseLicenseUrl: String,
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:24]:
```suggestion
    licenseUrl: String,
```

File: src/components/ImageDetails/ImageAttribution.vue [hunk 21]
```diff
@@ -21,13 +21,13 @@ export default {
...
     fullLicenseName: String,
     attributionHtml: String,
   },
   computed: {
-    licenseURL() {
-      return `${this.ccLicenseURL}&atype=rich`
+    licenseUrl() {
+      return `${this.openverseLicenseUrl}&atype=rich`
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:30]:
```suggestion
      return `${this.licenseUrl}&atype=rich`
```

File: src/components/ImageDetails/ImageAttribution.vue [hunk 22]
```diff
@@ -21,13 +21,13 @@ export default {
...
     image: {},
-    ccLicenseURL: String,
+    openverseLicenseUrl: String,
     fullLicenseName: String,
     attributionHtml: String,
   },
   computed: {
-    licenseURL() {
-      return `${this.ccLicenseURL}&atype=rich`
+    licenseUrl() {
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:29]:
```suggestion
    licenseUrlRichText() {
```

File: src/components/ImageDetails/ImageAttribution.vue [hunk 23]
```diff
@@ -3,12 +3,12 @@
...
       :full-license-name="fullLicenseName"
       :license="image.license"
-      :license-u-r-l="licenseURL"
+      :license-url="licenseUrl"
     />
     <CopyLicense
       :media="image"
       :full-license-name="fullLicenseName"
-      :license-u-r-l="licenseURL"
+      :license-url="licenseUrl"
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:11]:
```suggestion
      :license-url="licenseUrlRichText"
```

File: src/components/ImageDetails/ImageAttribution.vue [hunk 24]
```diff
@@ -3,12 +3,12 @@
     <MediaLicense
       :full-license-name="fullLicenseName"
       :license="image.license"
-      :license-u-r-l="licenseURL"
+      :license-url="licenseUrl"
```
Comment (obulat) [inline src/components/ImageDetails/ImageAttribution.vue:6]:
```suggestion
      :license-url="licenseUrlRichText"
```

File: src/components/ImageDetails/ImageInfo.vue [hunk 25]
```diff
@@ -63,7 +63,7 @@ export default {
   },
   props: [
     'image',
-    'ccLicenseURL',
+    'openverseLicenseUrl',
```
Comment (obulat) [inline src/components/ImageDetails/ImageInfo.vue:66]:
```suggestion
    'licenseUrl',
```

File: src/components/ImageDetails/PhotoDetails.vue [hunk 26]
```diff
@@ -126,7 +126,7 @@
           <ImageAttribution
             data-testid="image-attribution"
             :image="image"
-            :cc-license-u-r-l="openverseLicenseURL"
+            :openverse-license-url="openverseLicenseUrl"
```
Comment (obulat) [inline src/components/ImageDetails/PhotoDetails.vue:129]:
```suggestion
            :license-url="licenseUrl"
```

File: src/components/ImageDetails/PhotoDetails.vue [hunk 27]
```diff
@@ -141,7 +141,7 @@
           <ImageInfo
             data-testid="image-info"
             :image="image"
-            :cc-license-u-r-l="openverseLicenseURL"
+            :openverse-license-url="openverseLicenseUrl"
```
Comment (obulat) [inline src/components/ImageDetails/PhotoDetails.vue:144]:
```suggestion
            :license-url="licenseUrl"
```

File: src/components/ImageDetails/PhotoDetails.vue [hunk 28]
```diff
@@ -235,7 +235,7 @@ export default {
         ? getFullLicenseName(this.image.license, this.image.license_version)
         : ''
     },
-    openverseLicenseURL() {
+    openverseLicenseUrl() {
```
Comment (obulat) [inline src/components/ImageDetails/PhotoDetails.vue:238]:
```suggestion
    licenseUrl() {
```

File: src/components/MediaInfo/CopyLicense.vue [hunk 29]
```diff
@@ -75,7 +75,7 @@
           <template #license>
             <a
               class="photo_license"
-              :href="openverseLicenseURL"
+              :href="openverseLicenseUrl"
```
Comment (obulat) [inline src/components/MediaInfo/CopyLicense.vue:78]:
```suggestion
              :href="licenseUrl"
```

File: src/components/MediaInfo/CopyLicense.vue [hunk 30]
```diff
@@ -155,8 +155,8 @@
                     : $t('media-details.reuse.credit.copy-text')
                 }}
               </template>
-              <template v-if="openverseLicenseURL" #URL>
-                {{ openverseLicenseURL }}
+              <template v-if="openverseLicenseUrl" #URL>
```
Comment (obulat) [inline src/components/MediaInfo/CopyLicense.vue:158]:
```suggestion
              <template v-if="licenseUrl" #URL>
```

File: src/components/MediaInfo/CopyLicense.vue [hunk 31]
```diff
@@ -155,8 +155,8 @@
...
+                {{ openverseLicenseUrl }}
```
Comment (obulat) [inline src/components/MediaInfo/CopyLicense.vue:159]:
```suggestion
                {{ licenseUrl }}
```

File: src/components/MediaInfo/CopyLicense.vue [hunk 32]
```diff
@@ -186,7 +186,7 @@ export default {
     media: {},
     fullLicenseName: String,
     attributionHtml: String,
-    openverseLicenseURL: String,
+    openverseLicenseUrl: String,
```
Comment (obulat) [inline src/components/MediaInfo/CopyLicense.vue:189]:
```suggestion
    licenseUrl: String,
```

File: src/pages/audio/_id.vue [hunk 33]
```diff
@@ -67,7 +67,7 @@ const AudioDetailPage = {
     fullLicenseName() {
       return getFullLicenseName(this.audio.license, this.audio.license_version)
     },
-    openverseLicenseURL() {
+    openverseLicenseUrl() {
```
Comment (obulat) [inline src/pages/audio/_id.vue:70]:
```suggestion
    licenseUrl() {
```

File: src/pages/audio/_id.vue [hunk 34]
```diff
@@ -4,7 +4,7 @@
     <MediaReuse
       data-testid="audio-attribution"
       :media="audio"
-      :license-u-r-l="openverseLicenseURL"
+      :license-url="openverseLicenseUrl"
```
Comment (obulat) [inline src/pages/audio/_id.vue:7]:
```suggestion
      :license-url="licenseUrl"
```

File: src/pages/audio/_id.vue [hunk 35]
```diff
@@ -113,8 +113,8 @@ const AudioDetailPage = {
   methods: {
     ...mapActions([FETCH_AUDIO, FETCH_RELATED_MEDIA]),
     attributionHtml() {
-      const licenseURL = `${this.openverseLicenseURL}&atype=html`
-      return attributionHtml(this.audio, licenseURL, this.fullLicenseName)
+      const licenseUrl = `${this.openverseLicenseUrl}&atype=html`
```
Comment (obulat) [inline src/pages/audio/_id.vue:116]:
```suggestion
      const licenseUrl = `${this.licenseUrl}&atype=html`
```

File: src/utils/attribution-html.js [hunk 36]
```diff
@@ -1,4 +1,4 @@
-function attributionHtml(image, ccLicenseURL, fullLicenseName) {
+function attributionHtml(image, openverseLicenseUrl, fullLicenseName) {
```
Comment (obulat) [inline src/utils/attribution-html.js:1]:
```suggestion
function attributionHtml(image, licenseUrl, fullLicenseName) {
```

File: src/utils/attribution-html.js [hunk 37]
```diff
@@ -14,7 +14,7 @@ function attributionHtml(image, ccLicenseURL, fullLicenseName) {
   } else if (image.creator && !image.creator_url) {
     creator = `<span> by <span>${image.creator}</span></span>`
   }
-  const licenseLink = ` is licensed under <a href="${ccLicenseURL}" style="margin-right: 5px;">${fullLicenseName.toUpperCase()}</a>`
+  const licenseLink = ` is licensed under <a href="${openverseLicenseUrl}" style="margin-right: 5px;">${fullLicenseName.toUpperCase()}</a>`
```
Comment (obulat) [inline src/utils/attribution-html.js:17]:
```suggestion
  const licenseLink = ` is licensed under <a href="${licenseUrl}" style="margin-right: 5px;">${fullLicenseName.toUpperCase()}</a>`
```

File: src/utils/attribution-html.js [hunk 38]
```diff
@@ -27,7 +27,7 @@ function attributionHtml(image, ccLicenseURL, fullLicenseName) {
       .join('')
   }
 
-  const licenseImgLink = `<a href="${ccLicenseURL}" target="_blank" rel="noopener noreferrer" style="display: inline-block;white-space: none;margin-top: 2px;margin-left: 3px;height: 22px !important;">${licenseIcons}</a>`
+  const licenseImgLink = `<a href="${openverseLicenseUrl}" target="_blank" rel="noopener noreferrer" style="display: inline-block;white-space: none;margin-top: 2px;margin-left: 3px;height: 22px !important;">${licenseIcons}</a>`
```
Comment (obulat) [inline src/utils/attribution-html.js:30]:
```suggestion
  const licenseImgLink = `<a href="${licenseUrl}" target="_blank" rel="noopener noreferrer" style="display: inline-block;white-space: none;margin-top: 2px;margin-left: 3px;height: 22px !important;">${licenseIcons}</a>`
```

File: test/unit/specs/components/ImageDetails/copy-license.spec.js [hunk 39]
```diff
@@ -37,7 +37,7 @@ describe('CopyLicense', () => {
         creator: 'John',
         creator_url: 'http://creator.com',
       },
-      ccLicenseURL: 'http://license.com',
+      openverseLicenseUrl: 'http://license.com',
```
Comment (obulat) [inline test/unit/specs/components/ImageDetails/copy-license.spec.js:40]:
```suggestion
      licenseUrl: 'http://license.com',
```

File: test/unit/specs/components/ImageDetails/image-attribution.spec.js [hunk 40]
```diff
@@ -21,7 +21,7 @@ describe('ImageAttribution', () => {
         creator: 'John',
         creator_url: 'http://creator.com',
       },
-      ccLicenseURL: 'http://license.com',
+      openverseLicenseUrl: 'http://license.com',
```
Comment (obulat) [inline test/unit/specs/components/ImageDetails/image-attribution.spec.js:24]:
```suggestion
      licenseUrl: 'http://license.com',
```

File: test/unit/specs/components/ImageDetails/image-info.spec.js [hunk 41]
```diff
@@ -21,7 +21,7 @@ describe('Image Info', () => {
         creator: 'John',
         creator_url: 'http://creator.com',
       },
-      ccLicenseURL: 'http://license.com',
+      openverseLicenseUrl: 'http://license.com',
```
Comment (obulat) [inline test/unit/specs/components/ImageDetails/image-info.spec.js:24]:
```suggestion
      licenseUrl: 'http://license.com',
```

**Review by obulat — APPROVED** [review]

**Review by zackkrida — APPROVED** [review]

**Comment by zackkrida** [comment]
Thank you @theshakeabhi !

**Pull request merged**